azerothcore/azerothcore-wotlk
# fix(DB/quest_offer_reward_locale): Add ruRU localization to "An Aggressive Defense" quest

## Changes Proposed:
-  Add missing ruRU localization of quest_offer_reward_locale for "An Aggressive Defense" quest

## Issues Addressed:
- [Makes quest offer reward with ruRU localization](https://github.com/azerothcore/azerothcore-wotlk/issues/14148)

## SOURCE:
[Russian wowhead](https://www.wowhead.com/wotlk/ru/quest=1025/%D0%BB%D1%83%D1%87%D1%88%D0%B0%D1%8F-%D0%B7%D0%B0%D1%89%D0%B8%D1%82%D0%B0-%D0%BD%D0%B0%D0%BF%D0%B0%D0%B4%D0%B5%D0%BD%D0%B8%D0%B5)

## Tests Performed:
- Tested ingame

## How to Test the Changes:

1. .q add 1025
2. Complete the quest conditions
3. Start talking to the NPC to turn in the quest

 ![207247237-d1c94152-138b-4b05-984d-26aec834d0a6](https://user-images.githubusercontent.com/64589699/207256321-67c847de-0c01-452d-8ed1-823a731e98e3.png)

When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
